### ISSUE TITLE
External Video and SVG.

### ISSUE THREAD

**Issue by MrChuw** [anyproto/anytype-ts#934]
### Have you read a contributing guide?

- [X] I have read CONTRIBUTING.md
- [X] I have searched the existing requests and didn't find any that were similar
- [X] I have considered creating a pull request instead and want to proceed

### Clear and concise description of the problem

After being pointed to commit #929 in the forum and testing it a bit, it would be great to add support for videos and SVGs. I currently have 50GB of videos in my Obsidian vault and about 2GB of images, and I’m self-hosting Anytype. Since I share with my smartphone, I don’t need the videos on it, just on my work PC.

### Suggested solution

Basically the same thing that was done for external images, but for videos and SVGs.
